app/uk/gov/hmrc/rasapi/controllers/FileController.scala

```diff
@@ -50,7 +50,16 @@ class FileController @Inject()(
   val fileServe: String = "File-Read"
   private val _contentType: String = "application/csv"
 
-  def parseStringIdToBSONObjectId(id: String): Try[BSONObjectID] = BSONObjectID.parse(id)
+  def parseStringIdToObjectId(id: String): Try[ObjectId] = {
```

Comment:
this could be just:
```
Try {
  logger.info("log stuff")
  new ObjectId(id)
}
```
and the log could be popped into line 122 (which currently still refers to it as BSONObjectId)